<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FakturatorClass</class>
 <widget class="QMainWindow" name="FakturatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fakturator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="9" column="6">
     <widget class="QPushButton" name="exportButton">
      <property name="text">
       <string>Export</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Cena</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLineEdit" name="senderValue"/>
    </item>
    <item row="1" column="5">
     <widget class="QLineEdit" name="sumStrVal"/>
    </item>
    <item row="1" column="4">
     <widget class="QDoubleSpinBox" name="sumVal">
      <property name="suffix">
       <string>€</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLineEdit" name="factoryNumberVal"/>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Účet</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLineEdit" name="bankVal">
      <property name="inputMask">
       <string>0000</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Datum</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4">
     <widget class="QLineEdit" name="purposeVal"/>
    </item>
    <item row="4" column="4">
     <widget class="QLineEdit" name="formNumberVal"/>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Doklad č.</string>
      </property>
     </widget>
    </item>
    <item row="4" column="5">
     <widget class="QLineEdit" name="formNumberCodeVal"/>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Číslo faktúry</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Dodávateľ</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Účel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLineEdit" name="accountVal">
      <property name="inputMask">
       <string>00000000000000000000000000000000000000000000000000000000000000000</string>
      </property>
      <property name="maxLength">
       <number>65</number>
      </property>
     </widget>
    </item>
    <item row="8" column="2">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Splatnosť</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4">
     <widget class="QDateEdit" name="exVal"/>
    </item>
    <item row="7" column="4">
     <widget class="QDateEdit" name="dateVal"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionPridaj_odberate_a">
   <property name="text">
    <string>Pridaj odberateľa</string>
   </property>
  </action>
  <action name="actionNastav_svoje_info">
   <property name="text">
    <string>Nastav svoje info</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Ulož</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Načítaj</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="fakturator.qrc"/>
 </resources>
 <connections/>
</ui>
